parser_library/src/instruction_set_version.h

```diff
@@ -39,10 +39,29 @@ enum class instruction_set_version
     UNI
 };
 
-constexpr std::array<std::pair<std::string_view, instruction_set_version>, 23> instr_set_version_equivalents = { {
+constexpr std::array<std::pair<std::string_view, instruction_set_version>, 77> instr_set_version_equivalents = { {
```

Comment:
is using machine args in optable and vice versa actually supported?

could the definition be moved to a cpp file as a constinit extern const?

Reply:
1: Not it's not supported -> added machine variable

2: Moved to `assembler_options.cpp` and left as `constexpr`